src/write-yaml-template.ts

```diff
@@ -0,0 +1,45 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import { stringify } from 'json-to-pretty-yaml'
+
+const walk = (dir: string, prevResults?: string[]): string[] => {
+  const list = fs.readdirSync(dir)
+  const results = prevResults || []
+  list.forEach((file) => {
+    file = path.resolve(dir, file)
+    const stat = fs.statSync(file)
+    if (stat && stat.isDirectory()) {
+      walk(file, results)
+    } else {
+      results.push(file)
+    }
+  })
+  return results
+}
+
+;((): void => {
+  fs.mkdirSync('./bin/templates')
+  const jsonPaths = walk('./templates/json')
+  jsonPaths.forEach((jsonPath) => {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const json = require(jsonPath)
```

Comment:
You can avoid this eslint issue (and avoid the need to stringify the result) by reading the file using:
```suggestion
    const json = fs.readFileSync(jsonPath)
```

Reply:
Sorry, I must be half asleep.

I just realised that the line below is creating a YAML string from a JS object.

So in that case the above would need to become...
```
const json = JSON.parse(fs.readFileSync(jsonPath))
```